plugins/node/plugin.zsh

```diff
@@ -8,11 +8,17 @@ GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VE
 geometry_prompt_node_setup() {}
 
 geometry_prompt_node_render() {
-    if (( $+commands[node] )); then
-        GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
-        GEOMETRY_NPM_VERSION="$(npm --version 2> /dev/null)" 
-        echo "$GEOMETRY_NODE_NPM_VERSION $GEOMETRY_NODE_VERSION ($GEOMETRY_NPM_VERSION)"
+    local -a files
+    files=($(ls -a ${PWD} | egrep -e ".nvmrc|yarn.lock"))
```

Comment:
I would rather use `[[ -f $FILE ]]` instead of an array. It makes it easier to read and we are checking against files we already know the names:

    if [[ -f "$PWD/.nvmrc" || -f "$PWD/yarn.lock" ]]; then # if .nvmrc or yarn.lock exists...